lib/kasket/write_mixin.rb

```diff
@@ -40,7 +40,12 @@ def store_in_kasket(key = kasket_key)
       def kasket_keys(options = {})
         attribute_sets = [attributes]
 
-        previous_changes = options[:previous_changes] || previous_changes()
+        previous_changes = options[:previous_changes]
+        previous_changes ||= if respond_to?(:saved_changes)
+          saved_changes.transform_values(&:first)
+        else
+          previous_changes
```

Comment:
missing the `()` should be previous_changes()

Reply:
SEATTLE STYLE!